lib/dpl/providers/pages.rb

```diff
@@ -174,10 +197,18 @@ def sufficient_scopes?
         api.scopes.include?('public_repo') || api.scopes.include?('repo')
       end
 
-      def url_with_token
+      def remote_url
+        github_token? ? https_url_with_token : git_url
+      end
+
+      def https_url_with_token
         "https://#{github_token}@#{url}"
       end
 
+      def git_url
+        "git@#{url}.git"
```

Comment:
This pattern is incorrect. For SSH, Git remote URL looks like:
```
git@github.com:user/repo.git  # with a colon
# or
ssh://git@github.com/user/repo.git  # with a slash
```

Reply:
👍